packages/js/src/types.ts

```diff
@@ -1,7 +1,111 @@
-export interface PaginatedResponse<T = unknown> {
+export enum NotificationStatus {
+  READ = 'read',
+  SEEN = 'seen',
+  UNREAD = 'unread',
+  UNSEEN = 'unseen',
+}
+
+export enum NotificationButton {
+  PRIMARY = 'primary',
+  SECONDARY = 'secondary',
+}
+
+export enum NotificationActionStatus {
+  PENDING = 'pending',
+  DONE = 'done',
+}
+
+export enum ActorType {
```

Comment:
This enum is convoluted. Is its purpose to drive the icon selection?

the none status should be replaced with the null type. The as far as I can tell there is a user vs system actor. 

Id also argue that it might be more helpful to replace it with an icon property.

Reply:
I'm not sure either if it's useful to the users at all, like they are just interested in the actor icon URL and nothing more.
But that's just what we have rn and we can improve later.

Reply:
https://linear.app/novu/issue/COM-34/improve-the-actor-interface